snapshots/devmapper/snapshotter.go

```diff
@@ -38,9 +39,12 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
+type fsType string
+
 const (
 	metadataFileName = "metadata.db"
 	fsTypeExt4       = "ext4"
+	fsTypeXfs        = "xfs"
```

Comment:
nits:
* specify the `fsType` type that is defined above
* XFS is typically written as XFS or xfs, not Xfs

```suggestion
	fsTypeExt4 fsType = "ext4"
	fsTypeXFS  fsType = "xfs"
```

Reply:
Done